src/index.js

```diff
@@ -230,6 +245,7 @@ export default class Form extends Component {
         validateOn,
         customErrorProp: errorProp = 'error',
         keepErrorOnFocus,
+        errors: propsErrors = {}
```

Comment:
At this point, we have a small problem when a `null` value is received because [JS default parameters only work with `undefined` values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/default_parameters). I suggest an older approach for this:
```suggestion
...
        errors,
      } = this.props
      const propsErrors = errors || {}
...
```

Reply:
Great point! Thanks.